src/clients/catalog.ts

```diff
@@ -113,6 +114,25 @@ export class Catalog extends JanusClient {
     })
   }
 
+  public search(
+    query: string,
+    authMethod: AuthMethod = 'AUTH_TOKEN',
```

Comment:
I think this should be `STORE_TOKEN` no? In fact, it's a public endpoint

Reply:
Yess, good catch!